web/src/pages/Checkout/index.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+
+import { Location } from './components/Location'
+import { Order } from './components/Order'
+import { Payment } from './components/Payment'
+import { 
+    Navbar, Main, Title, SubTitle, Div,
+} from './styled'
+import { Logo } from '../../components/Logo'
+import { Button } from '../../components/Button'
+
+export const Checkout = () => (
+    <>
+        <Navbar>
+            <Logo />
+        </Navbar>
+        <Main>
+            <Div>
+                <Title>Finalizar seu pedido</Title>
+                <SubTitle>Endereço</SubTitle>
+                <Location />
+                <SubTitle>Pague pelo site</SubTitle>
+                <Payment />
+                <Button style={{  width: '28vw' }}>Finalizar pedido</Button>
```

Comment:
Se eu entendi bem. Este botão deve ocupar 100% da tela... talvez `width: 100%;` seja melhor nesse caso, se o tamanho desse card alterar no futuro.

Se não for isso, desconsidera este comentário kkkk

Reply:
Não... O @olucasmoro configurou os outros componentes dessa coluna pra terem esse valor no width. Eu apenas peguei isso do CSS dele e coloquei no `Button` compartilhado.
Se fosse o caso dele ocupar 100%, poderia passar a prop `full`, que já faz esse papel